src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -113,7 +119,7 @@ public JestElasticsearchClient(Map<String, String> props) {
   }
 
   // visible for testing
-  protected JestElasticsearchClient(Map<String, String> props, JestClientFactory factory) {
+  public JestElasticsearchClient(Map<String, String> props, JestClientFactory factory) {
```

Comment:
I don't see how this change is used in this PR. Did I miss a change?

Reply:
Residual from a different strategy -- thanks for catching